docs/assets/extra.css

```diff
@@ -66,25 +68,34 @@
 }
 
 .md-typeset table:not([class]) th {
-    background-color: rgba(0,184,212,.1);
+    background-color: rgba(0, 184, 212, 0.1);
 }
 
 .md-typeset table:not([class]) tbody tr:nth-child(even) td {
-    background-color: rgba(0,184,212,.05);
+    background-color: rgba(0, 184, 212, 0.05);
 }
 
 /* Display of get-a-demo link */
 
 .getademo {
-    display: inline-block;
-    padding: 0.4rem;
-    height: 1.8rem;
     background-color: #f39200;
-    color: #fff;
-    vertical-align: middle;
-    font-size: 0.9rem;
-    font-weight: 700;
+    border-radius: 999px;
+    box-shadow: #f39200 0 10px 20px -10px;
     box-sizing: border-box;
+    color: #ffffff;
+    cursor: pointer;
+    font-size: 0.7rem;
+    font-weight: 700;
+    line-height: 24px;
+    opacity: 1;
+    outline: 0 solid transparent;
```

Comment:
In my opinion we should allow an outline on links when the user navigate by keyboard. This is an accessibility feature, all other links on the page have it, why should we remove it from one of the most important links?